src/pretalx/orga/templates/orga/speaker/form.html

```diff
@@ -42,6 +43,11 @@ <h2 class="d-flex justify-content-between align-items-baseline">
             {{ proposal_title }})
         </span>
         <div class="ml-auto mb-1">
+            {% if can_mark_speaker and accepted_submissions.count and not form.instance.has_arrived %}
+                <a class="btn btn-info" href="{{ form.instance.orga_urls.toggle_arrived }}">
+                    {% translate "Mark speaker as arrived" %}
```

Comment:
It would probably be good to show the speaker’s `has_arrived` status here if they have arrived already, as otherwise the button *and* information will just … not be there.

This should also be a full toggle button (ie also mark as unarrived), as allowing users to revert an accidental click would be good.